cli_tools/gce_vm_image_import/importer/finisher.go

```diff
@@ -0,0 +1,47 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"context"
+)
+
+// finisher represents the second (and final) phase of import. For bootable
+// disks, this means translation and publishing the final image. For data
+// disks, this means publishing the image.
+type finisher interface {
```

Comment:
`finisher` sounds like it's just responsible for wrapping things up (which, in case of data disks, is true, kind of), but for bootable disks, there's more to in the `finishing` stage than in the actual disk import. 

Can you think of a more appropriate name to reflect the fact this process can take time and is very involved? `processor`, `transformer`, `adapter`?

Reply:
Cool, changed to `processor`.

More info:
I struggled with naming, since there are two different paths to represent:

* dataDisk -> publish
* bootableDisk -> transform, publish

I think your idea of `processor` is more descriptive.